tests/integration/conftest.py

```diff
@@ -18,3 +20,25 @@ def test_env():
     with open(config_file) as f:
         env = yaml.safe_load(f)
     return env
+
+
+@pytest.fixture(scope="session")
+def create_new_request(test_env):
```

Comment:
Could you rename this to something that describes what is being returned? For example you could name it `default_request` or something like that.

I'm sorry I missed this in my previous review.